### DISCUSSION TITLE
Verify email by sending a digit code instead of a verification link

### DISCUSSION THREAD

**Discussion by baptisteArno** [supabase/supabase#1949] [Feature Requests]
A modern way of verifying a user's email address is by sending a digit code by email:

![CleanShot 2021-06-13 at 09 07 27](https://user-images.githubusercontent.com/16015833/121798461-ccb26900-cc26-11eb-8999-3a7e6535cf77.png)
☝ Taken from Plausible, one of the best sign-up processes I ever stumbled upon.

Pros:
- Easier for the user: can check up his phone
- Easier for the developer: no need to handle redirection on the email link click

Cons:
- ?

Any way to implement this kind of solution with the current lib state? 😃

**Comment by silentworks** [comment]
At the moment this can probably be done by turning off email verification and setting up your own code to handle this. But it will probably be easier to do once workflows are released.

One con I see in the solution above is that the onus would be on the developer to implement this page inside of their app.

**Reply by weepy** [reply]
you also probably need to add some expiry into that.
